packages/next/server/web/spec-extension/fetch-event.ts

```diff
@@ -1,7 +1,33 @@
 import { DeprecationError } from '../error'
-import { FetchEvent } from '../spec-compliant/fetch-event'
 import { NextRequest } from './request'
 
+const responseSymbol = Symbol('response')
+const passThroughSymbol = Symbol('passThrough')
+export const waitUntilSymbol = Symbol('waitUntil')
+
+class FetchEvent {
```

Comment:
**nit:** we could merge this `FetchEvent` class inside `NextFetchEvent`.

Given `FetchEvent` was previously coming from another file, inheritance made sense.
But now we that `../spec-compliant` is gone, and given we don't export `FetchEvent`, that inheritance is not really needed.

Reply:
This will be addressed in a separate PR 👍